dist/adaptive-html.cjs.js

```diff
@@ -382,10 +382,19 @@ rules.strong = {
     });
   }
 };
+rules.iframe = {
+  filter: 'iframe',
+  replacement: function replacement(content, node) {
+    var fallbackText = 'To view this content, please open this video in the Guru app';
+    return wrap(createTextBlock(fallbackText), {
+      style: 'emphasis'
+    });
+  }
+};
 rules.image = {
   filter: 'img',
   replacement: function replacement(content, node) {
-    var alt = node.alt || '';
+    var alt = node.getAttribute('alt') || '';
     var src = node.getAttribute('src') || '';
```

Comment:
are these values ever changing?

Reply:
nope! i can update them to `const` in `adaptivecard-rules.js`